opal/group.go

```diff
@@ -293,6 +317,47 @@ func resourceGroupUpdateAuditMessageChannels(ctx context.Context, d *schema.Reso
 	return nil
 }
 
+func resourceGroupUpdateResources(ctx context.Context, d *schema.ResourceData, client *opal.APIClient) diag.Diagnostics {
+	var rawResources []any
+	if resourceI, ok := d.GetOk("resource"); ok {
+		rawResources = resourceI.(*schema.Set).List()
+	}
+
+	resourcesWithAccessLevel := make([]opal.ResourceWithAccessLevel, 0, len(rawResources))
+	for _, rawResource := range rawResources {
+		resource := rawResource.(map[string]any)
+		var accessLevelRemoteIDPtr *string
+		accessLevelRemoteID := resource["access_level_remote_id"].(string)
+		if accessLevelRemoteID != "" {
+			accessLevelRemoteIDPtr = &accessLevelRemoteID
+		}
+
+		resourcesWithAccessLevel = append(resourcesWithAccessLevel, opal.ResourceWithAccessLevel{
+			ResourceId:          resource["id"].(string),
+			AccessLevelRemoteId: accessLevelRemoteIDPtr,
+		})
+	}
+	tflog.Debug(ctx, "Setting group resources", map[string]any{
+		"id":        d.Id(),
+		"resources": resourcesWithAccessLevel,
+	})
+
+	updateInfo := opal.UpdateGroupResourcesInfo{
+		Resources: resourcesWithAccessLevel,
+	}
+
+	if _, err := client.GroupsApi.SetGroupResources(ctx, d.Id()).UpdateGroupResourcesInfo(updateInfo).Execute(); err != nil {
+		var gErr *opal.GenericOpenAPIError
+		if errors.As(err, &gErr) {
+			log.Println("error string", string(gErr.Body()))
+		} else {
+			log.Println("not", err)
+		}
```

Comment:
i think this can be deleted entirely - i see that i did this in owner.go/resource.go too so it'd be nice to fix those

(my guess is that I wrote this to debug while writing `diagFromErr` but missed deleting this)

Reply:
Thanks, removed.